src/js/view/controls/controls.js

```diff
@@ -141,7 +141,9 @@ export default class Controls extends Events {
         }
 
         // Floating Close Button
-        const floatingConfig = model.get('floating');
+        let floatingConfig = model.get('floating');
+        floatingConfig = floatingConfig && floatingConfig.disabled ? null : floatingConfig;
+
         if (floatingConfig) {
```

Comment:
What if the logic in this block was a function that returned early when `floatingConfig.disabled` is true? I'm thinking that would avoid repeating the `floatingConfig` check (covered once in `if`) and the ternary operators (just exit when `floatingConfig.disabled`).